utils/constants.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+// These strings are prefixed onto data put in the backend, to designate its type.
+const (
+	XML_PREFIX  = "xml"
+	JSON_PREFIX = "json"
+)
+
+// The following numeric constants serve as configuration defaults
+const (
+	CASSANDRA_DEFAULT_TTL_SECONDS = 2400
+	REDIS_DEFAULT_EXPIRATION      = 60
```

Comment:
Good catch. It'll be minutes. This value gets multiplied by 60 downstream. On a second look, the constant was declared and not used. I was thinking about changing the `yaml` configuration field's name from `"backend.redis.expiration"` to `"backend.redis.expiration_minutes"` but this would constitute a breaking change so I didn't. Let me know if you agree with the correction.

Reply:
I'm good with this change. Yeah I agree I don't think it is worth introducing a breaking change but changing it internally to indicate the units is helpful.